compiler/rustc_ast_lowering/src/item.rs

```diff
@@ -212,28 +211,6 @@ impl<'hir> LoweringContext<'_, 'hir> {
     pub fn lower_item(&mut self, i: &Item) -> Option<hir::Item<'hir>> {
```

Comment:
This function always returns `Some`.

Reply:
I know, but it's not my function and I'm no longer touching that part of the code.

Reply:
ok